test/analysis/test_cytoscape.py

```diff
@@ -6,17 +6,23 @@
 import spras.config as config
 from spras.analysis.cytoscape import run_cytoscape
 
-config.init_from_file("config/config.yaml")
+config.init_from_file("input/config.yaml")
```

Comment:
if using the config file in the input folder in analysis, make sure to set cytoscape include to true incase